.github/workflows/scripts/start-acs.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/bash
```

Comment:
I appreciate your effort to refactor the steps into a separate script.

Consider [this](https://github.com/stackrox/stackrox/blob/master/.github/workflows/scripts/create-release.sh) as a good example for the usual structure of such a script.

Reply:
Done